compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -636,7 +648,10 @@ void addCPULinalgExtTileAndVectorizePipeline(
     GenericVectorizationPassOptions options;
     options.useConfiguredVectorSizes = pipelineOpt.useConfiguredVectorSizes;
     options.enableVectorMasking = pipelineOpt.enableVectorMasking;
+    options.enableCleanup = false;
```

Comment:
wait, why?

Reply:
`false` was introduced because of the legacy issues in GPU pipeline. I'd prefer not pull this in CPU backends, because it took me some time to make CPU pipelines consistent.

Reply:
Sure, the only reason I added this because I was anyway running the canonicalizer after vectorization, so duplication of running canonicalization patterns seemed weird. Removing this now.

Reply:
Removed this now.